service/src/main/kotlin/app/cash/backfila/ui/PathBuilder.kt

```diff
@@ -0,0 +1,69 @@
+package app.cash.backfila.ui
+
+import app.cash.backfila.ui.pages.ServiceShowAction
+import app.cash.protos.backfila.ui.SortCol
+
+// TODO add tests!
+data class PathBuilder(
```

Comment:
For some other services, it's been helpful to have our own class so that we can take enums or other types as inputs and encode it in a sane way into the URL. Otherwise, I agree `HttpUrl.Builder` would be a great choice.